### ISSUE TITLE
WSL Path is incorrect with Custom built Distro

### ISSUE THREAD

**Issue by artman41** [wslutilities/wslu#138]
**Background Information:**
 - Build: 19041
 - WSL: Fedora 31
 - Version: 3.1.1

 WSLU Version
```
[artman41@DESKTOP-TQGCI07 portfolio_site ]$ yum list installed | grep wslu
wslu.noarch                                  3.1.1-1                             @copr:copr.fedorainfracloud.org:wslutilities:wslu
```

Windows Version
![image](https://user-images.githubusercontent.com/5568965/89122481-5a79df80-d4bf-11ea-92b7-3b514e97924a.png)

**Describe the bug**
`wslusc` appears to generate an incorrect path to `wsl.exe`

```
[artman41@DESKTOP-TQGCI07 bin ]$ ll
total 168
-rw-r--r-- 1 artman41 artman41   136 Jul 27 11:45 appletviewer.policy
-rwxr-xr-x 1 artman41 artman41   224 Jul 27 11:45 format.sh
-rwxr-xr-x 1 artman41 artman41 26660 Jul 27 11:45 fsnotifier
-rwxr-xr-x 1 artman41 artman41 32888 Jul 27 11:45 fsnotifier64
-rw-r--r-- 1 artman41 artman41   506 Jul 27 11:45 idea64.vmoptions
-rw-r--r-- 1 artman41 artman41  5462 Jul 27 11:45 idea.png
-rw-r--r-- 1 artman41 artman41 11281 Jul 27 11:45 idea.properties
-rwxr-xr-x 1 artman41 artman41  8173 Jul 27 11:45 idea.sh
-rw-r--r-- 1 artman41 artman41  2317 Jul 27 11:45 idea.svg
-rw-r--r-- 1 artman41 artman41   514 Jul 27 11:45 idea.vmoptions
-rwxr-xr-x 1 artman41 artman41   299 Jul 27 11:45 inspect.sh
-rw-r--r-- 1 artman41 artman41 39520 Jul 27 11:45 libdbm64.so
-rw-r--r-- 1 artman41 artman41  2613 Jul 27 11:45 log.xml
-rwxr-xr-x 1 artman41 artman41   646 Jul 27 11:45 printenv.py
-rwxr-xr-x 1 artman41 artman41   808 Jul 27 11:45 restart.py
[artman41@DESKTOP-TQGCI07 bin ]$ wslusc -g idea.sh
[info] Create shortcut idea.sh.lnk successful
[artman41@DESKTOP-TQGCI07 bin ]$ pwd
/home/artman41/idea/current/bin
[artman41@DESKTOP-TQGCI07 bin ]$ wslusc -i `pwd`/idea.png -g `pwd`/idea.sh
[info] You choose to use custom icon: /home/artman41/idea/current/bin/idea.png. Processing...
[info] Converting png icon to ico...
[info] Create shortcut idea.sh.lnk successful
```
![Annotation 2020-08-01 233844](https://user-images.githubusercontent.com/5568965/89111644-3df28e80-d450-11ea-8fb3-27ed2a7269e0.jpg)

**To Reproduce**
- Add a Custom Distro (in this case, I used [Fedora 31](https://gist.github.com/artman41/858450e2c29b239d8692213b684ada25))
- Create an Installation of IntelliJ IDEA Community
- Run `wslusc`

**Expected behavior**
The path to be correct & idea to start

**WSL Distros**
```
[artman41@DESKTOP-TQGCI07 portfolio_site ]$ wsl.exe -l
Windows Subsystem for Linux Distributions:
fedora (Default)
Ubuntu
docker-desktop
docker-desktop-data
```

**Comment by patrick330602** [comment]
Which version of wslu you are using and which windows build of windows are you using? Need to narrow down the problem

**Comment by artman41** [comment]
WSLU Version
```
[artman41@DESKTOP-TQGCI07 portfolio_site ]$ yum list installed | grep wslu
wslu.noarch                                  3.1.1-1                             @copr:copr.fedorainfracloud.org:wslutilities:wslu
```

Windows Version
![image](https://user-images.githubusercontent.com/5568965/89122481-5a79df80-d4bf-11ea-92b7-3b514e97924a.png)

I've added them to the issue :+1:

**Comment by artman41** [comment]
@patrick330602 I've also updated the issue with my wsl distros

```
[artman41@DESKTOP-TQGCI07 portfolio_site ]$ wsl.exe -l
Windows Subsystem for Linux Distributions:
fedora (Default)
Ubuntu
docker-desktop
docker-desktop-data
```

**Comment by patrick330602** [comment]
Thank you for adding the report. I will look into the problem and will release the fix as part of 3.2.0.

**Comment by patrick330602** [comment]
Just commited the fix. It is currently working will be released as part of wslu 3.2.0
